src/Sql/Sql/OutboundFirewallRules/Cmdlet/GetAzureSqlServerOutboundFirewallRule.cs

```diff
@@ -0,0 +1,99 @@
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.OutboundFirewallRules.Model;
+using Microsoft.Rest.Azure;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.OutboundFirewallRules.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzSqlServerOutboundFirewallRules cmdlet
+    /// </summary>
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlServerOutboundFirewallRule", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.None)]
+    [OutputType(typeof(AzureSqlServerOutboundFirewallRulesModel))]
+    public class GetAzureSqlServerOutboundFirewallRule : AzureSqlServerOutboundFirewallRulesCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the allowed FQDN of the Azure Sql Database Server Outbound Firewall Rule
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "The Azure Sql Database Server Outbound Firewall Rule FQDN.")]
+        [Alias("Name")]
+        [ValidateNotNullOrEmpty]
+        [SupportsWildcards]
+        public string AllowedFQDN { get; set; }
+
+        /// <summary>
+        /// Gets an allowed FQDN from the list of outbound firewall rules from the service.
+        /// </summary>
+        /// <returns>A single Firewall Rule</returns>
+        protected override IEnumerable<AzureSqlServerOutboundFirewallRulesModel> GetEntity()
+        {
+            List<AzureSqlServerOutboundFirewallRulesModel> results = null;
+
+            if (this.MyInvocation.BoundParameters.ContainsKey("AllowedFQDN") && !WildcardPattern.ContainsWildcardCharacters(AllowedFQDN))
+            {
+                Model.AzureSqlServerOutboundFirewallRulesModel model;
+                results = new List<AzureSqlServerOutboundFirewallRulesModel>();
+
+                try
+                {
+                    model = ModelAdapter.GetOutboundFirewallRule(this.ResourceGroupName, this.ServerName, this.AllowedFQDN);
+                    results.Add(model);
+                }
+                catch (CloudException ex)
+                {
+                    if (ex.Response.StatusCode == System.Net.HttpStatusCode.NotFound)
+                    {
+                        throw new PSArgumentException(
+                            string.Format(Microsoft.Azure.Commands.Sql.Properties.Resources.ServerOutboundFirewallRuleFQDNDoesNotExist, this.AllowedFQDN, this.ServerName), "AllowedFQDN");
```

Comment:
```suggestion
                            string.Format(Resources.ServerOutboundFirewallRuleFQDNDoesNotExist, this.AllowedFQDN, this.ServerName), "AllowedFQDN");
```
Believe we can use Resources directly.

Reply:
Done. Fixed.